documentation/docs/developer_guide/obsolete_and_deprecated_features/deprecated_features.md

```diff
@@ -0,0 +1,36 @@
+# Deprecated features for CABLE developers #
+
+Coding with the deprecated developer features described here is discouraged as they are scheduled to become obsolete
+and removed from future versions of the CABLE code. 
```

Comment:
We could add:

Development that includes any deprecated or obsolete feature will require modifications before being accepted for a CABLE release.

Reply:
Ok I added this bit